yt/loaders.py

```diff
@@ -1292,9 +1295,15 @@ def load_sample(fn=None, progressbar: bool = True, timeout=None, **kwargs):
         `None` means "no limit". This parameter is directly passed to down to
         requests.get via pooch.HTTPDownloader
 
-    Any additional keyword argument is passed down to `yt.load`.
-    Note that in case of collision with predefined keyword arguments as set in
+    Notes
+    -----
+    - Any additional keyword argument is passed down to `yt.load`.
+    - In case of collision with predefined keyword arguments as set in
     the data registry, the ones passed to this function take priority.
+    - Datasets with slashes '/' in their names can safely be used even on Windows.
```

Comment:
did not know this...

Now what if we named a dataset `CON`?  :)

Reply:
> did not know this...

Well it is only true _with_ to the present PR :)

> Now what if we named a dataset CON? :)

I don't get it :(

Reply:
https://help.interfaceware.com/v6/windows-reserved-file-names